src/PrestaShopBundle/Translation/Provider/TranslationFinder.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * 2007-2019 PrestaShop SA and Contributors
+ * 2007-2019 PrestaShop and Contributors
```

Comment:
It's "PrestaShop SA and Contributors" ([see devdocs](https://devdocs.prestashop.com/1.7/development/coding-standards/#license-information))

Reply:
This is not what is being computed by the UpdateLicensesCommand
https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Command/UpdateLicensesCommand.php#L40

Are you sure the devdocs is right and the code is wrong ? If yes, we need to update all license headers for both 1.7.6.x and develop 😞

Reply:
Yep I'm positive

Reply:
Issue created https://github.com/PrestaShop/PrestaShop/issues/15116